app/controllers/votes_controller.rb

```diff
@@ -1,27 +1,34 @@
 class VotesController < ApplicationController
   def new
-    @n = 1
     first = Place.first
     last = Place.last
     #need to include something to test if the place id actually exists
     rand_place = rand(first.id..last.id)
     @place = Place.find_by_id(rand_place)
     @photo_array = searchFlickr(@place)
-
+    @n = rand(0..@photo_array.length)
```

Comment:
`@n` won't be very clear to the next person editing this file..or the view its available in